doc/dev/workflow.md

```diff
@@ -0,0 +1,63 @@
+# Overview
+
+This document describes the general developement workflow for
+developing and working with Mixer in the broader context of Istio.
+This document includes information on building Mixer properly and
+then building the associated containers in a local environment
+prior to prow updating the istio/istio dependency repositories.
+
+## Prepare tree
+
+Prepare the Mixer repository and Go environment variables as
+specified in the 
+[contribution guidelines](https://github.com/istio/istio/blob/master/CONTRIBUTING.md).
+
+## Prepare a registry account on gcr.io
+
+Follow the
+[Google Container Registry Quickstart](https://https://cloud.google.com/container-registry/docs/quickstart).
+
+## Using the codebase
+
+Follow the
+[Using the code base docuementation](https://github.com/istio/istio/blob/master/devel/README.md#using-the-code-base://github.com/istio/istio/blob/master/devel/README.md).
```

Comment:
typo in "documentation"

Reply:
this is duplicated in README.md, and this work is being PR'ed on issue https://github.com/istio/istio/issues/1060.

Reply:
fixed by merge into https://github.com/istio/istio/issues/1060.